### DISCUSSION TITLE
fine adjusting 2monitors

### DISCUSSION THREAD

**Discussion by Yamaha3001** [i3/i3#5632] [Q&A] [answered]
Hi,
I have two monitors:
first: 2560x1440
second:1080x1920
my problem is not left or right, but to fine adjust the second mointor a bit higher.
Can I adjust this in i3/config file? If yes how?
or:
Is there an application I need to download?

**Comment by waterkip** [comment, marked as answer]
Have you tried it with autorandr?

**Comment by BirdeeHub** [comment]
there is a --pos flag

**Reply by BirdeeHub** [reply]
in xrandr
